scan_rr.go

```diff
@@ -1216,6 +1217,115 @@ func (rr *DS) parse(c *zlexer, o string) *ParseError      { return rr.parseDS(c,
 func (rr *DLV) parse(c *zlexer, o string) *ParseError     { return rr.parseDS(c, o, "DLV") }
 func (rr *CDS) parse(c *zlexer, o string) *ParseError     { return rr.parseDS(c, o, "CDS") }
 
+func (rr *IPSECKEY) parse(c *zlexer, o string) *ParseError {
+	for i := 0; i < 3; i++ {
```

Comment:
I think this might be cleaner if we did something like:
```go
for _, out := range []*uint8{&rr.Precedence, &rr.GatewayType, &rr.Algorithm} {
```

and then below:
```go
*out = uint8(num)
```

Reply:
Or it might be better just to spell it out one-by-one honestly.